.github/jobs/macos_cmake.yml

```diff
@@ -0,0 +1,14 @@
+parameters:
+  vmImage: ''
+
+steps:
+  - script: |
+      brew uninstall cmake
```

Comment:
Looks like this should workaround https://github.com/BabylonJS/BabylonNative/issues/683, thanks!

Reply:
Same for ninja which is also on homebrew